style.css

```diff
@@ -71,24 +71,48 @@ header {
     align-items: center;
     justify-content: center;
     margin-top: 200px;
-    border-radius: 25px;  
+    border-radius: 25px; 
+    display: block;
+   
 }
 
 .text h1 {
     margin-top: -50px;
     margin-bottom: 50px;
     background: #56c3e4;
     border-radius: 32px;
+
 }
 
-.features, #container1, #container2, #container3 {
+.features{
     background: rgb(140, 140, 184);  
     border-radius: 50px;
 }
 
-#txt1, #txt2, #txt3 {
-    font-size: 20px;
-    margin-top: -60px;
```

Comment:
The text describing the features isn't centered horizontally and vertically. You need to center the text in th div for a better look

Reply:
Read about flex. It makes centering items very easy.